### DISCUSSION TITLE
can't use wallpaper from the store

### DISCUSSION THREAD

**Discussion by shiin07** [Taiizor/Sucrose#101] [Q&A]
I can't use the wallpapers i downloaded from the store, but I can use the wallpapers already in the library by default. I changed all my performance settings to resume already. plz help

**Comment by Taiizor** [comment]
I understand, you're using a very old version of Sucrose. You can verify this information yourself in the image below.

![Sucrose-Wallpaper-Engine-Store](https://github.com/user-attachments/assets/1004d36f-9169-458a-a474-52c7134e5cdb)

Please right-click on the Sucrose system tray icon in the taskbar and select 'Check for Updates' from the menu. A visual guide is provided below.

![Sucrose-Wallpaper-Engine-System-Tray](https://github.com/user-attachments/assets/1e44026e-24e0-4eec-96f6-d3888e22f087)

If you encounter an error while checking for updates or are unable to update, you will need to manually install the latest version. To do this, go to [this address](https://github.com/Taiizor/Sucrose?tab=readme-ov-file#download) and download the appropriate version for your system from GitHub. If you're unsure which to choose, you can select the one marked in the image below.

![Sucrose-Wallpaper-Engine-Download-GitHub](https://github.com/user-attachments/assets/196b2765-1a73-41e4-b430-eaf22d318bb1)

HINT: Your system is most likely x64 architecture, but if it is x86, choose the appropriate version.

**Reply by shiin07** [reply]
PS: also i didn't make myself clear the first time i can download the wallpapers from the store its just that when i used them they just appear black

**Reply by Taiizor** [reply]
Every video has a format (codec). Not all 'Video Plugins' in Sucrose support these formats. To resolve this, if you select the video plugin as 'MpvPlayer' instead of 'Nebula' within Sucrose, the black screen issue will be resolved.


Since 'MpvPlayer' generally supports all video formats, you can use it as the default. However, for 'Nebula,' you can resolve your issue by installing video extensions via the Microsoft Store. Below are links for these extensions, and all of the mentioned extensions are Microsoft products, so you can use them safely.

AV1 Video Extension: https://www.microsoft.com/store/productId/9MVZQVXJBQ9V
VP9 Video Extensions: https://www.microsoft.com/store/productId/9N4D0MSMP0PT
Web Media Extensions: https://www.microsoft.com/store/productId/9N5TDP8VCMHS
MPEG-2 Video Extension: https://www.microsoft.com/store/productId/9N95Q1ZZPMH4
HEVC Video Extensions from Device Manufacturer: https://www.microsoft.com/store/productId/9N4WGH0Z6VHQ

After installing these, the black screen issue for the mentioned video will be resolved.

**Reply by Taiizor** [reply]
@shiin07, has your black screen issue with Nebula been resolved after installing the packages above?

**Reply by shiin07** [reply]
after installing everything, nebula still don't work for me mpv player does though? do you recommned me using mpv instead of nebula?

**Reply by Taiizor** [reply]
May I know which theme from the store you tried to run on Nebula?

Of course, I recommend MpvPlayer, but your choices here will only make a difference in resource consumption. Normally, it would be better to choose the one that suits your system best.

**Comment by shiin07** [comment]
thanks for the response! i am using the latest version as i just installed sucrose i actually found out the problem myself it seems that the default video and gif player doesn't work for me and so i had to change the default player into webview and it worked, i am just curious why the default players didn't work for me.

**Comment by Taiizor** [comment]
Are you using Windows Insider? If so, take a look at [this issue](https://github.com/Taiizor/Sucrose/issues/79).

If you're not an Insider user, are the players that aren't working Xavier for GIFs and Nebula for Videos?

**Reply by shiin07** [reply]
i'm not using windows insider i'm using windows 11 23h2, yes xavier and nebula still doesn't work for me

**Reply by Taiizor** [reply]
Could you share the GIF file that you are unable to play on Xavier with me here?
